docs/guidance/kuberay-with-MCAD.md

```diff
@@ -14,83 +14,101 @@ MCAD allows you to deploy Ray cluster with a guarantee that sufficient resources
 - Support for multiple Kubernetes clusters; dispatching jobs to any one of a number of Kubernetes clusters.
 
 
-In order to queue Ray cluster(s) and `gang dispatch` them when aggregated resources are available please refer to the setup [KubeRay-MCAD integration](https://github.com/IBM/multi-cluster-app-dispatcher/blob/quota-management/doc/usage/examples/kuberay/kuberay-mcad.md) with configuration files [here](https://github.com/IBM/multi-cluster-app-dispatcher/tree/quota-management/doc/usage/examples/kuberay/config).
+In order to queue Ray cluster(s) and `gang dispatch` them when aggregated resources are available please refer to the setup [KubeRay-MCAD integration](https://github.com/IBM/multi-cluster-app-dispatcher/blob/quota-management/doc/usage/examples/kuberay/kuberay-mcad.md) on a Kubernetes Cluster or an OpenShift Cluster.
```

Comment:
This url should be updated to refer to the CodeFlare repository instead of `IBM`